### PR TITLE
Finished Homework2

### PR DESCRIPTION
**Pull request by HKalininQA**
Hello, I've finished my HomeWork from the second lesson.

### CODE REVIEW

File: HW2_1.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Eleks = ['John', 'Mary', 'Anna', 'Max', 'Igor', 'Serhii', 'Oleksandr']
+Toshiba = ['John', 'Max', 'Oksana', 'Sasha', 'Anna', 'Alex', 'Nikolai']
+
+Toshiba.extend(Eleks)
+print(Toshiba)
```
Comment (bohdandovbsyh) [inline HW2_1.py]: after your manipulations, Eleks should not have employees. Please fix it

File: HW2_3.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+omnivores = ['John', 'Mary', 'Anna', 'Mark']
+vegetarians = ['Josef', 'Julia', 'Hamood', 'Habibi']
+guests = []
+
+omnivores.extend(vegetarians)
```
Comment (bohdandovbsyh) [inline HW2_3.py]: you change the original data. It is not good

File: HW2_3.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+omnivores = ['John', 'Mary', 'Anna', 'Mark']
+vegetarians = ['Josef', 'Julia', 'Hamood', 'Habibi']
+guests = []
```
Comment (bohdandovbsyh) [inline HW2_3.py]: looks like it was never used.

File: HW2_4.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+VIP = ["Henrich", "Marcus", "Joseph", "Ulrich"]
```
Comment (bohdandovbsyh) [inline HW2_4.py]: i can add new VIP guest. But the number should be fixed

File: HW2_5.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+names = ['John Doe', 'John Dow', 'Mary Sued', 'Mark Sue', 'Max Doe', 'Mary Sued', 'John Doe', 'Bob Hill', 'Bob Hill']
+
+names_unique = []
+
+for name in names:
+    if name not in names_unique:
+        names_unique.append(name)
+
+print(names_unique)
```
Comment (bohdandovbsyh) [inline HW2_5.py]: as I mentions in the lesson this HW should be done only using functions of types which were shown

File: HW2_4.py (hunk 4 above)
Comment (HKalininQA) [reply HW2_4.py]: Not very clear from your comment what should be changed.

**Comment by HKalininQA** [comment]
Hi @bohdandovbsyh I've fixed your comments above please check the latest commits.

File: HW2_4.py (hunk 4 above)
Comment (bohdandovbsyh) [reply HW2_4.py]: just make it as immutable type. tuple for example of frozenset

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**